scripts/liquidation_bot/deploy.ts

```diff
@@ -34,6 +34,7 @@ async function main() {
   const assets = await Promise.all(Array(numAssets).fill(0).map((_, i) => comet.getAssetInfo(i)));
 
   const liquidator = await dm.deploy(
+    'liquidator',
```

Comment:
Oh interesting this is using the deployment manager but not a deployment 🤔

Reply:
it uses a deployment, right? it needs to know `network` + `deployment` to find the correct instance of Comet

Reply:
So the deployment is in `scripts/liquidation_bot`? I guess the deploy script can be anywhere if its not a base, just hadn't really thought about this. So you can't run sims on it but you can add to roots/aliases 🤯 Kinda cool I think it all works out with the cli tool too because roots is always updated non-destructively now